output/json/json_test.go

```diff
@@ -47,7 +47,7 @@ func getValidator(t *testing.T, expected []string) func(io.Reader) {
 				t.Errorf("Read unexpected line number %d, expected only %d entries", i, len(expected))
 				continue
 			}
-			assert.Equal(t, expected[i-1], string(s.Bytes()))
+			assert.JSONEq(t, expected[i-1], string(s.Bytes()))
```

Comment:
It checks just if the object contains the same keys and values but not the order as before. If you think the order here is relevant then we can restore it.